x-pack/plugins/lists/common/schemas/request/import_exceptions_schema.mock.ts

```diff
@@ -23,6 +23,10 @@ export const getImportExceptionsListSchemaMock = (
   type: 'detection',
 });
 
+/** 
+  This mock holds the old properties of the Exception List item
+  so that we can test the migration test cases
```

Comment:
Might be helpful to define "old" here since we don't have API versioning

Reply:
please find [0e5f52f](https://github.com/elastic/kibana/pull/152757/commits/0e5f52f2e5f361024dfd130f1939d8ea285759d5)